dummy/src/main/java/com/ss/android/ugc/aweme/feed/model/Aweme.java

```diff
@@ -3,16 +3,34 @@
 //Dummy class
 public class Aweme {
```

Comment:
Stub classes should go into the "dummy" package. This package is compiled against the dummy package but the dummy package classes are not shaded into this one.

Reply:
I don't get it. Do you mind being more specific?

Reply:
Currently, your class is in the main module. The dummy module is located here:

![image](https://github.com/ReVanced/revanced-integrations/assets/13122796/c9108925-0fdd-4301-924c-467ee97f8dbc)

Reply:
![image](https://github.com/ReVanced/revanced-integrations/assets/43563783/6cdba325-9eb7-49e1-864b-b8eece84468a)
No matter how many times I check, I still saw that class under dummy module.

Reply:
Well thats my bad, for some reason I did not notice it, thanks for noting.